src/cdk-experimental/dialog/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare var module: {id: string};
```

Comment:
A note on this one: I still can't find a way to get Bazel to pick up the `typings.d.ts` that is one level up in `cdk-experimental`. This is also something that came up in https://github.com/angular/material2/pull/11481.

Reply:
That's expected; Bazel doesn't allow referencing source files that aren't in the package's subtree; if you wanted to reference a common `d.ts` file, you have to create a build rule for it and depend on that.